testsuite/MDAnalysisTests/analysis/test_hole.py

```diff
@@ -141,6 +156,33 @@ def test_plot3D_rmax(self, H):
         assert isinstance(ax, mpl_toolkits.mplot3d.Axes3D), "H.plot3D(rmax=float) did not produce an Axes3D instance"
 
 
+@pytest.mark.skipif(executable_not_found("hole"), reason="Test skipped because HOLE not found")
+class TestHOLEtraj_default(_TestHOLEtraj):
+    pass
+
+@pytest.mark.skipif(executable_not_found("hole"), reason="Test skipped because HOLE not found")
+class TestHOLEtraj_dynamic_cpoint(_TestHOLEtraj):
+
+    @pytest.fixture()
+    def reference(self):
```

Comment:
This is the same reference as above. Does it have to be repeated in the class?

Reply:
Probably not, I might well only have been starting to get the hang of using pytest fixtures.

Reply:
Actually, it is _not_ the same reference. The values are subtly different for "mean HOLE rxncoord", "minimum radius", and "min_radius" because the CPOINT parameter was varied. The test trajectory does not move very much so it does not make a big difference.

I'll leave it as it is.